### PR TITLE
Create my first draft PR

### PR DESCRIPTION
**Pull request by juanm-roa**
## Context
We are creating our first PR
## Objective
To create the first PR and practice
## Details
First Issue and PR

### CODE REVIEW

**Review by CristhianRodriguezMolina — CHANGES_REQUESTED** [review]
Bad very bad you can use pattern matching here

**Review by Wilzhar — CHANGES_REQUESTED** [review]
You can use pattern matching here

**Review by CristhianRodriguezMolina — CHANGES_REQUESTED** [review]
New line at the end of the file

File: facil.exs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Make sure to start the backend server too to connect the client:
+[PECO - A platform to teach Computational thinking (CT) - Server](https://github.com/CristhianRodriguezMolina/plataforma-ct-backend)
+
+The admin credentials to login are set in the server repository
+
+If you are using the default values for environment variables you can login using the next admin credentials:
+```
+User: 214896
+Password: Expedited-Wrongful3-Skewer
+```
```
Comment (CristhianRodriguezMolina) [inline facil.exs:91]: Doxeable

File: README.md [hunk 2]
```diff
@@ -1,2 +1,91 @@
...
+  npm start
+```
+
+
+## Use the platfom
+
+Now you can use a web browser and navigate to `localhost:3000`
+
+Make sure to start the backend server too to connect the client:
+[PECO - A platform to teach Computational thinking (CT) - Server](https://github.com/CristhianRodriguezMolina/plataforma-ct-backend)
```
Comment (Wilzhar) [inline README.md:83]: @juanm-roa please remove this.

**Review by Wilzhar — CHANGES_REQUESTED** [review]

File: facil.exs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+
+
+## Authors
+
+- [@CristhianRodriguezMolina](https://github.com/CristhianRodriguezMolina)
+- [@Wilzhar](https://github.com/Wilzhar)
+
+
+## Requirements
```
Comment (CristhianRodriguezMolina) [inline facil.exs:27]: Hey yo

**Review by CristhianRodriguezMolina — CHANGES_REQUESTED** [review]

File: facil.exs (hunk 1 above)
Comment (Wilzhar) [reply facil.exs:91]: what does doxeable mean?

**Review by CristhianRodriguezMolina — APPROVED** [review]
:100: Melo

**Review by Wilzhar — APPROVED** [review]

**Pull request merged**